Add: clos topology

Brianiiix opened the pull request.

Brianiiix committed:
Add: clos topology

tony2037 commented on main.py:
```diff
@@ -1,7 +1,7 @@
 from mininet.net import Mininet
 from mininet.cli import CLI
-
+from topologies.clos import CLOS
```
You should use conditional import to make the code concise and flexible
Eg. 
if args.topo = 'clos':
  from topologies.clos import CLOS as CustomTopo
else:
  from topologies.ocs import OCS as CustomTopo

tony2037 commented on topologies/clos.py:
```diff
@@ -1,5 +1,32 @@
-    def build( self ):
-        pass
\ No newline at end of file
+class CLOS(Base):
```
To obey the style, it should be Clos

tony2037 replied:
it should be
class Clos( Base ):

tony2037 committed:
Fix: Fix the inflexibility between different topologies

Introduce argsparse and flexibility for different topologies
Specify arguments to test different topologies

[Test]
sudo python3 main.py --topo ocs
sudo python3 main.py --topo clos
sudo python3 main.py --topo ocs --cli
